WeakAuras/RegionTypes/RegionPrototype.lua

```diff
@@ -473,6 +473,21 @@ function WeakAuras.regionPrototype.AddExpandFunction(data, region, cloneId, pare
     WeakAuras.Animate("display", data, "main", data.animation.main, region, false, nil, true, cloneId);
   end
 
+  function region:OptionsClosed()
+    region:EnableMouse(false)
+    region:SetScript("OnMouseDown", nil)
+  end
+
+  function region:ClickToPick()
+    region:EnableMouse(true)
+    region:SetScript("OnMouseDown", function()
+      WeakAuras.PickDisplay(region.id, nil, true)
+    end)
+    if region.GetFrameStrata and region:GetFrameStrata() == "TOOLTIP" then
```

Comment:
Is this still needed?

Reply:
Yes, without the strata change when i make a cursor aura with frameStrata = "TOOLTIP" visible from WeakAurasOptions i can't click anything anymore.

```
!nwvBZjUnq4)oPDAdMq6LJBM(bGdUl3qaMiFV1jJbf71G6jKCLK5L8b)BV7UM364CZ0egXJF8Us7(S7ksANeNiwNiYOpi0zdYGYAsMeLi2nnp3dbgNAnzk6nE0Sa9S26q4K28NB41o86T0kAISmSY6MwCHxRvoh62ZjI9xS3oyjAs8(cireGDHshcKMu07zwLb9R)044PpmE4imeFwdMShSzOn9hpCY7Xih8s0jzaODwAuRRZb0yFq6ceiJEpsopWhJhOeYJNh)OXAao(Kk2no7EKYJdgKw6dwuNcoPXRXtk(1PhvAste54kDyxv4SlDG3)Bvl4i5BhbFhbzGoi)2rW3)LQpvTO(FTnvQrlXvOAXFwTOD1Iwvl(89ZKoWeE3hGWW8CapJnGGm6QMoVd349SZO1dkDEsm9Cj8sRDaQCg08Qf)6HtS2XJpv9juUrbJW7zfrQlwjp0uGYupMC)zI(mbd)adzF3XWD1fCQldyyPhMFs9s6)tluyXjxzu(v)VlMOhLoDIyviu4FxRwBLlTxRST(4p0UOSbz5TOCj9y35PwLlpjeJWA2WfnN3BcGlxMcp1lld9)PVcYF0dJk)tZSBbhdFaYuYNQzP176qnkQLlbNVEYj(ebbjPi36wtQaHL6TY9(GRK1LsdSbm1Jv2ZsqIaZIdVzW5zuHrUgom05lFMnquMNR2LiMpONiEUiU3J0HsPNOql3JbHWxaA99zhCKcc(99pVjZCaVjIzdhporSemGtLkwz3ofJfp)DpUX8y5a11qPNMsFgdG65MpoC8SrFEmFiMZsalYzk)FZZp0wi166I0giU2mEYFYV3oQU4G1bJu)fus5BTAFl1oSEdLmTVo66i6rD(HRrgmCs8Whr3YOrxAKGVylxT8qD3HQgnmhK1D68DqJi264wm4XHdNWXyko5aZ)p3xws77y1vFD02)5L7V9VseBofxNgBq0kqTCfE13B6EDx8V30(T)rNOBU70Cbk3ARKVpwzw6SLf8T0L6G6OirLgQHWrghWXltOHnPAP33GflXPniDwn0GmtH9lPiZ(M2JT(niRdJon4ZRhYAWxSP4Ne51030GxsnWWRKtQxAeoi9wvwyvYK7W2GOOoVTBu3UTVH)PjSD89WgS(qDmmvY)o
```

Reply:
That aura is using custom code to attach to the mouse frame. Using the standard and supported way to attach it to the cursor just happens to work perfectly.